server/routers/userRouter.js

```diff
@@ -14,27 +14,24 @@ userRouter.route("/get/").get(async (req, res) => {
 userRouter.post(
   "/post/",
   expressAsyncHandler(async (req, res, err) => {
-
     // Some Code in here
     try {
       if (await userModel.exists({ user_email: req.body.user_email })) {
         return res.sendStatus(400).send("user already exist");
```

Comment:
if that is can u check creating new account with already existed user cause u didn't check the `sendStatus` to `status` here.